docs_user/modules/proc_migrating-the-rgw-backends.adoc

```diff
@@ -115,11 +114,7 @@ rgw.rgw.cephstorage-1.qynkan  cephstorage-1  172.17.3.26:8090  running (16s)
 rgw.rgw.cephstorage-2.krycit  cephstorage-2  172.17.3.81:8090  running (13s)
 ----
 
-. Ensure that the new RGW backends are reachable on
-the new ports, because you are going to enable an IngressDaemon on port 8080
-later in the process. For this reason, log in to each RGW node (the {CephCluster}
-nodes) and add the iptables rule to allow connections to both 8080 and 8090
-ports in the {CephCluster} nodes.
+. Ensure that the new RGW back ends are reachable on the new ports. You are eventually going to enable an IngressDaemon on port 8080. Log in to each {CephCluster} node that includes RGW and add the `iptables` rule to allow connections to both 8080 and 8090 ports in the {CephCluster} nodes:
```

Comment:
"Ensure that the new RGW back ends are reachable on the new ports. You are eventually going to enable an IngressDaemon on port 8080."
does this mean
"Ensure that the new RGW back ends are reachable on the new ports, so you can enable an IngressDaemon on port 8080." ?
or no?

Maybe "later" is better than "eventually"?

Reply:
@fmount What do you think about changing the sentence to: "Ensure that the new RGW back ends are reachable on the new ports, so you can enable an IngressDaemon on port 8080 later." ?

Reply:
+1 from me, thank you!